File: snippets/java/app/src/main/java/finetuning/ListEvents.java
/* (C)2024 */
package finetuning;

import com.cohere.api.Cohere;
import com.cohere.api.resources.finetuning.finetuning.types.ListEventsResponse;

public class ListEvents {
    public static void main(String[] args) {
        Cohere cohere = Cohere.builder().clientName("snippet").build();

        ListEventsResponse response = cohere.finetuning().listEvents("test-id");

        System.out.println(response);
    }
}

File: snippets/node/finetuning/list-events.js
import { CohereClient } from 'cohere-ai';

const cohere = new CohereClient({ token: process.env.COHERE_API_KEY! });

(async () => {
  const events = await cohere.finetuning.listEvents('test-finetuned-model-id');

  console.log(events);
})();
